modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase

```diff
@@ -823,6 +823,87 @@ definition {
 		ObjectPortlet.viewEntry(entry = "Entry B");
 	}
 
+	@description = "LPS-151676 - Verify that it's possible to edit the relationships of the Custom Object entry (One-to-Many)"
+	@priority = "4"
+	test CanEditOneToManyRelationshipOfCustomObjectEntries {
+		task ("Add a Custom Object, a field, create a 1toM relationship with the User object, and add an entry (via API)") {
```

Comment:
Given some Custom Object entries that contain a Custom Field and Relationship (One-to-many)